### DISCUSSION TITLE
[Feature] Toggle panes in the PDF/EPUB/HTML reader tabs

### DISCUSSION THREAD

**Discussion by mjthoraval** [windingwind/zotero-actions-tags#166] [Ideas]
I would like to have a shortcut to toggle the left and right panes in the PDF/EPUB/HTML reader taba, similar to the one in the My Library tab done in https://github.com/windingwind/zotero-actions-tags/discussions/118.

**Comment by windingwind** [comment]
https://github.com/windingwind/zotero-actions-tags/discussions/169

**Comment by windingwind** [comment]
Need 1.0.0-19 or higher
